packages/plugin-loyalties-ui/src/configs/scoreCampaign/components/Form.tsx

```
@@ -0,0 +1,397 @@
+        <Attributes>
+          <>
+            <li>
+              <b>{__("Attributions")}</b>
+            </li>
+            {attributes.map((attribute) => {
+              const { label = "", value } = attribute || {};
+
+              if (!value) {
+                return null;
+              }
+              return <li onClick={() => onClick(value, close)}>{__(label)}</li>;
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)